x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/components/total_charts.test.tsx

```diff
@@ -10,39 +10,38 @@ import '../../../__mocks__/engine_logic.mock';
 
 import React from 'react';
 
-import { shallow, ShallowWrapper } from 'enzyme';
+import { mount, ReactWrapper } from 'enzyme';
 
-import { EuiButtonTo } from '../../../../shared/react_router_helpers';
+import { EuiButtonEmptyTo } from '../../../../shared/react_router_helpers';
 import { AnalyticsChart } from '../../analytics';
 
 import { TotalCharts } from './total_charts';
 
 describe('TotalCharts', () => {
-  let wrapper: ShallowWrapper;
-
+  let wrapper: ReactWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>;
   beforeAll(() => {
     jest.clearAllMocks();
     setMockValues({
       startDate: '1970-01-01',
       queriesPerDay: [0, 1, 2, 3, 5, 10, 50],
       operationsPerDay: [0, 0, 0, 0, 0, 0, 0],
     });
-    wrapper = shallow(<TotalCharts />);
+    wrapper = mount(<TotalCharts />);
```

Comment:
Haha I regret previously telling you to blanket change to `mount` now :) In this test I'd like to request we switch back to `shallow` and instead stop asserting on `h4`/`EuiButtonEmptyTo` text. In this scenario we should leave that testing in the hands of the new `DataPanel` component, now that it exists.

I'm not being super specific right now since it's getting late, sorry, but please let me know if you'd like to pair on this.

Reply:
https://github.com/elastic/kibana/pull/92939/commits/9782526e982dd2ca32d8630efca8ef59d5d479fb